frontend/admin-client/package.json

```diff
@@ -31,6 +31,7 @@
     "@angular/compiler-cli": "~10.2.4",
     "@types/jasmine": "~3.5.0",
     "@types/jasminewd2": "~2.0.3",
+    "@types/jquery": "^3.5.5",
```

Comment:
Why do we need types/jquery?

Reply:
Bootstrap's included code uses jquery which I'm leveraging to close the modal.